components/SectionRoadmap.vue

```diff
@@ -6,23 +6,24 @@
           Roadmap
         </div>
         <div class="section-title tm-rf7 tm-bold tm-lh-title">
-          <span class="percentage">{{ progressTotal }}%</span> complete
+          <!-- <span class="percentage">{{ progressTotal }}%</span> complete -->
+          <span class="percentage">100%</span> complete
         </div>
         <div class="section-milestones">
           <div class="section-milestones__title tm-rf3 tm-bold tm-lh-title">
-            Milestones
+            Releases
           </div>
           <div class="section-milestones__cta">
             <tm-button
               to-link="external"
-              href="https://github.com/orgs/cosmosdevs/projects/1"
+              href="https://github.com/cosmos/cosmos-sdk/releases/tag/v0.40.0"
```

Comment:
This should stay the same URL – but maybe change the link text to "View project ↗️". The URL for the Cosmos SDK item on the right was meant to be updated with this release URL.